unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/Tests/CharacterControllerTests.cs

```diff
@@ -384,16 +384,27 @@ void OnMessageFromEngine(string type, string payload)
             WebInterface.OnMessageFromEngine += OnMessageFromEngine;
             yield return null;
             WebInterface.OnMessageFromEngine -= OnMessageFromEngine;
-            
+
             Assert.IsTrue(rotationMatch);
         }
-        
+
         [UnityTest]
         public IEnumerator CharacterTeleportOnVariableChange()
         {
             yield return InitCharacterPosition(50, 2, 0);
             DataStore.i.player.lastTeleportPosition.Set(Vector3.zero);
             UnityEngine.Assertions.Assert.AreApproximatelyEqual(0, Vector3.Distance(DCLCharacterController.i.characterPosition.worldPosition, Vector3.zero), 2.0f);
         }
+
+        [Test]
+        public void CharacterContainsCorrectGroundLayers()
+        {
+            var groundLayerMask = DCLCharacterController.i.groundLayers.value;
+            Assert.IsTrue((groundLayerMask
```

Comment:
It was going to return true all the times otherwise
```suggestion
            Assert.IsTrue((0
```

Reply:
I tested that implementation yesterday and it was working fine but let me check if there's a test case I may have not taken into account and change it to your way

Reply:
Yeah the old way only tested if the target layers were in the layermask but not if there were extra layers, [changed it](https://github.com/decentraland/unity-renderer/pull/3743/commits/712df68b71bd1a7c46a21a9c95fd41ea59289bc3) to your sugestion, thanks!